engine/src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -60,7 +61,7 @@ trait CromwellApiService extends HttpService with PerRequestCreator with Service
       get {
         Try(WorkflowId.fromString(workflowId)) match {
           case Success(w) =>
-            requestContext => perRequest(requestContext, MetadataBuilderActor.props(serviceRegistryActor), GetStatus(w))
+            requestContext => perRequest(requestContext, metadataBuilderProps, GetStatus(w))
```

Comment:
Since they're all basically the same you could collapse these even further with
```
def handleMetadataRequest(req) = requestContext => perRequest(requestContext, metadataBuilderProps, req)
```
and then in the API calls, e.g:
```
case Success(w) => handleMetadataRequest(GetStatus(w))
```